Source/diablo.cpp

```diff
@@ -144,6 +145,8 @@ static void diablo_parse_flags(int argc, char **argv)
 			SetPrefPath(argv[++i]);
 		} else if (strcasecmp("--config-dir", argv[i]) == 0) {
 			SetConfigPath(argv[++i]);
+		} else if (strcasecmp("--ttf-dir", argv[i]) == 0) {
+			SetTtfPath(argv[++i]);

```

Comment:
The flag is called `--ttf-dir` but actually sets the full path to the TTF font.
Shouldn't it set the directory only?